CPT331.Data.Migration/Migrations/M0002.cs

```diff
@@ -265,9 +265,9 @@ public override void Up()
 				.Row(new { Name = "Theft And Related Offences (Other Than Mv)" })
 				.Row(new { Name = "Theft Of Motor Vehicle Parts Or Contents" })
 				.Row(new { Name = "Threatening Behaviour" })
-				.Row(new { Name = "Tins" })
 				.Row(new { Name = "Traffic And Related Offences" })
 				.Row(new { Name = "Traffic And Vehicle Regulatory Offences" })
+				.Row(new { Name = "Traffic Infringements" })
```

Comment:
I'm happy with this, however we'll need to rebuild the DB... the changed order will throw out the current IDs